packages/genesys-spark-components/src/components/beta/gux-blank-state-beta/tests/gux-blank-state-beta.e2e.playwright.ts

```diff
@@ -0,0 +1,137 @@
+          </gux-blank-state-beta>
+        `
+      );
+
+      const primaryMessage = page.getByTestId('primary-message');
+      await expect(primaryMessage).toBeVisible();
+      await expect(primaryMessage).toHaveText('Test primary message');
+    });
+
+    test('should render image slot content', async ({ page }) => {
+      await setContent(
+        page,
+        `
+          <gux-blank-state-beta>
+            <gux-icon slot="image" icon-name="bot" decorative="true" data-testid="image-icon"></gux-icon>
+            <div slot="primary-message">Test message</div>
+          </gux-blank-state-beta>
+        `
+      );
+
+      const imageIcon = page.getByTestId('image-icon');
+      await expect(imageIcon).toBeVisible();
+    });
+
+    test('should render additional guidance slot content', async ({ page }) => {
+      await setContent(
+        page,
+        `
+          <gux-blank-state-beta>
+            <div slot="primary-message">Test message</div>
+            <div slot="additional-guidance" data-testid="guidance">Additional guidance text</div>
+          </gux-blank-state-beta>
+        `
+      );
+
+      const guidance = page.getByTestId('guidance');
+      await expect(guidance).toBeVisible();
+      await expect(guidance).toHaveText('Additional guidance text');
+    });
+
+    test('should render call to action slot content', async ({ page }) => {
+      await setContent(
+        page,
+        `
+          <gux-blank-state-beta>
+            <div slot="primary-message">Test message</div>
+            <button slot="call-to-action" data-testid="cta-button">Primary Action</button>
+          </gux-blank-state-beta>
+        `
+      );
+
+      const ctaButton = page.getByTestId('cta-button');
+      await expect(ctaButton).toBeVisible();
+      await expect(ctaButton).toHaveText('Primary Action');
+    });
+  });
+
+  test.describe('properties', () => {
+    test('should apply left alignment class when alignment is set to left', async ({
+      page
+    }) => {
+      await setContent(
+        page,
+        `
+          <gux-blank-state-beta alignment="left">
+            <div slot="primary-message">Left aligned</div>
+          </gux-blank-state-beta>
+        `
+      );
+
+      const component = page.locator('gux-blank-state-beta');
+      const container = component.locator('.gux-container');
+      await expect(container).toHaveClass(/gux-alignment-left/);
+    });
+
+    test('should apply no padding class when noPadding is true', async ({
+      page
+    }) => {
+      await setContent(
+        page,
+        `
+          <gux-blank-state-beta no-padding>
+            <div slot="primary-message">No padding</div>
+          </gux-blank-state-beta>
+        `
+      );
+
+      const component = page.locator('gux-blank-state-beta');
+      const container = component.locator('.gux-container');
+      await expect(container).toHaveClass(/gux-no-padding/);
+    });
+  });
+
+  test.describe('interactions', () => {
```

Comment:
I think this is the only test other than the render confi ones that you need. The others can be added to the render config if they are not already there.